src/components/Tabs/renderTab.tsx

```diff
@@ -16,16 +16,27 @@ export interface RenderTabProps {
 
 export const renderTab =
   ({ onClick, currentTab, showCounter, page, context }: RenderTabProps) =>
+  // Linter doesn't understand that switch processes all cases
+  // eslint-disable-next-line consistent-return
   (tab: Tab, i: number) => {
     const { useRouter, handlerDecorator } = context;
     const router = useRouter();
 
-    if (isURL(tab?.ref)) {
-      return renderLinkTab({
-        ...useLink({ router, handlerDecorator }, { href: tab.ref, text: tab.title }),
-        i,
-      });
+    const type = tab.type;
+
+    if (!(type === 'group' || type === 'link')) {
+      return null;
     }
```

Comment:
```suggestion
    if (!['group', 'link'].includes(type)) {
      return null;
    }
```

вот тут я еще не совсем понимаю. Либо здесь проверка лишняя, либо в свиче дефоолт лишний, ведь другого type кроме как group и link у нас не могут быть

Reply:
Если использовать запись, которую ты предложил, type refinement не сработает